docs/reference.rst

```diff
@@ -112,6 +112,8 @@ Providers
     parsl.providers.TorqueProvider
     parsl.providers.KubernetesProvider
     parsl.providers.PBSProProvider
+    parsl.providers.ExecutionProvider
+    parsl.Providers.ClusterProvider
```

Comment:
This PR is failing in the documentation checking stage of CI testing. It looks like it is because this class doesn't exist, which I think is because you have a capitalized P in providers in this line.